jbpm-console-ng-case-mgmt/jbpm-console-ng-case-mgmt-client/src/main/java/org/jbpm/console/ng/cm/client/comments/CaseCommentsPresenter.java

```diff
@@ -0,0 +1,208 @@
+                    @Override
+                    public void callback(List<CaseCommentSummary> comments) {
+                        boolean editing =false;
+                        String updateActionLabel;
+                        for (CaseCommentSummary caseCommentSummary : comments) {
+                            CaseCommentAction deleteCommentAction = null;
+                            updateActionLabel=null;
+                            if(identity.getIdentifier().equals(caseCommentSummary.getAuthor())){
+                                if (caseCommentSummary.getId().equals(currentUpdatedCommentId)) {
+                                    editing = true;
+                                    updateActionLabel = translationService.format(SAVE);
+                                } else {
+                                    editing = false;
+                                    updateActionLabel = translationService.format(EDIT);
+                                }
+
+                                deleteCommentAction=new CaseCommentAction() {
+
+                                    @Override
+                                    public String label() {
+                                        return translationService.format(DELETE);
+                                    }
+
+                                    @Override
+                                    public void execute() {
+                                        deleteCaseComment(caseCommentSummary.getId());
+                                    }
+                                };
+                            }
+
+                            caseCommentView.addComment(editing, updateActionLabel, caseCommentSummary.getId(), caseCommentSummary.getAuthor(),
+                                        caseCommentSummary.getText(), caseCommentSummary.getAddedAt(), deleteCommentAction);
+                        }
+                    }
+                }
+        ).getComments(serverTemplateId, containerId, caseId, 0, 10);
+    }
+
+    protected void addCaseComment(String caseCommentText) {
+        if ("".equals(caseCommentText.trim())) {
+            caseCommentView.displayNotification(translationService.format(CASE_COMMENT_CANT_BE_EMPTY));
+        } else {
+            CaseCommentSummary caseCommentSummary = new CaseCommentSummary();
+            caseCommentSummary.setCaseId(caseId);
+            caseCommentSummary.setText(caseCommentText);
+            caseCommentSummary.setAuthor(identity.getIdentifier());
+            addCaseComment(caseCommentSummary);
+        }
+    }
+
+    protected void addCaseComment(final CaseCommentSummary caseCommentSummary) {
+        caseService.call(
+                new RemoteCallback<Void>() {
+                    @Override
+                    public void callback(Void response) {
+                        refreshComments();
+                    }
+                },
+                new DefaultErrorCallback()
```

Comment:
usage of DefaultErrorCallback should be avoided, there is global a error handler in place now.